en/qgep_swmm/Check-SWMM-File.rst

```diff
@@ -93,3 +93,22 @@ Missing population densities
 ``qgep_od.catchment_area.population_density_current`` or ``qgep_od.catchment_area.population_density_planned`` are not filled.
 
 
+Subcatchment and rain Gages
+---------------------------
+If you want to attach every subcatchments to a single rain gage, edit the file with SWMM interface:
```

Comment:
If you want to attach every subcatchments to a single rain gauge, edit the file with SWMM interface: